docs/source/development_faq.rst

```diff
@@ -1,19 +0,0 @@
-.. _development_faq:
```

Comment:
Yes, but we only use it for alpha/beta. Currently, I plan to use the FAQ to capture q&a that later may be integrated to other parts of the docs. This section will also expand to address some changes over the next 6 months and summer intern project work.

Although your suggestion is reasonable, let's leave it as a FAQ of one now. ":one: is the loneliest number..." I forget the song that these lyrics were used in. I don't anticipate the FAQ to stay at one entry for long ;-)